src/Illuminate/Foundation/Console/EnvironmentCommand.php

```diff
@@ -40,6 +40,9 @@ class EnvironmentCommand extends Command
      */
     public function handle()
     {
-        $this->line('<info>Current application environment:</info> <comment>'.$this->laravel['env'].'</comment>');
+        $this->components->info(sprintf(
+            'The application is in the [%s] environment.',
```

Comment:
The application isn't necessarily _in_ a specific environment, rather the environment is set to a specific value. For example, I could set `APP_ENV` to `production` on my local system.

```suggestion
            'The application environment is [%s].',
```

Reply:
I also think putting the value in the middle of a sentence is harder to mentally parse, I like this change since it returns it to the end of the line

Reply:
[Done.](https://github.com/laravel/framework/pull/43065/commits/62b2dce73285595105a80879e45dcb559ca3587b)